docs/chapters/architecture/DLT/diagnostics-log-and-tracing.rst

```diff
@@ -0,0 +1,181 @@
+:orphan:
+
+.. _diagnostics-log-and-tracing:
+
+
+Diagnostics Log and Tracing
+===========================
+
+Introduction and goals
+----------------------
+The Diagnostic, Log and Tracing (DLT) daemon provides a log and trace interface in ECUs. The standardized protocol for logging inside ECU is specified in the AUTOSAR 4.0 DLT, and GENIVI has an open source project `dlt-daemon`_ that follows those standards.
+
+Besides dlt-daemon, GENIVI offers `dlt-viewer`_ which is another software that allows a client to connect to the dlt-daemon and receive logs remotely.
+
+PELUX has decided to integrate GENIVIs' DLT component which also works with dlt-viewer.
```

Comment:
I'm missing a reason why. I think the way our arch documentation should be phrased should include a "why" when we have chosen a certain solution, even if it was the only solution we were looking at. Most likely we still have a reason why?